prepare-image.sh

```diff
@@ -104,5 +104,8 @@ if [[ -n "${PATCH_LIST:-}" ]]; then
 fi
 rm -f /bin/patch-image.sh
 
+# purge pip cache
+python3 -m pip cache purge
```

Comment:
How many places do we call pip in? If not many, it would be even better to call pip install with `--no-cache-dir` to not create any caches to begin with.

Reply:
yeah, I was actually going to change it to that as we run pip only twice and only when we install ironic related packages